utils/node_local_test.py

```diff
@@ -1694,8 +1693,8 @@ def run_dfuse(server, conf):
     container = str(uuid.uuid4())
     dfuse.start(v_hint='no_pool')
     print(os.statvfs(dfuse.dir))
-    subprocess.run(['df', '-h'])
-    subprocess.run(['df', '-i', dfuse.dir])
+    subprocess.run(['df', '-h'])  # nosec
+    subprocess.run(['df', '-i', dfuse.dir]) # nosec
```

Comment:
(pylint-subprocess-run-check) Using subprocess.run without explicitly set `check` is not recommended.